src/main.js

```diff
@@ -1,5 +1,24 @@
 // query selector variables go here 👇
 
+var posterTitle = document.querySelector('.poster-title');
+var posterQuote = document.querySelector('.poster-quote');
+var posterImg = document.querySelector('.poster-img');
+var showRandom = document.querySelector('.show-random');
```

Comment:
Is this a button? When a var is defined as a verb, it indicates that it is an action the user would take on it. I would add Btn to the variable to clarify. Example: `showRandomPosterBtn`